RetailCoder.VBE/Navigation/RegexSearchReplace/RegexSearchReplace.cs

```diff
@@ -96,6 +96,11 @@ private List<RegexSearchResult> SearchSelection(string searchPattern)
         {
             using (var pane = _vbe.ActiveCodePane)
             {
+                if (pane == null || pane.IsWrappingNullReference)
```

Comment:
As per the discussion in the chat, we shouldn't bother with null checks; checking the `IsWrappingNullReference` should be sufficient since we presume that all wrappers, including the `ICodePane`, will never return a `null`. Same comment apply to all occurrences in this PR.

Reply:
IIRC the wrappers override the `==` operator to make such null checks entirely equivalent to checking for `IsWrappingNullReference` though.